### PR TITLE
(FM-6924) - Change windows user to Administrator in tests

### PR DESCRIPTION
**Pull request by pmcmaw**

### CODE REVIEW

File: spec/acceptance/init_spec.rb [hunk 1]
```diff
@@ -1,15 +1,21 @@
 # run a test task
 require 'spec_helper_acceptance'
 
+username = if fact('operatingsystem') == 'windows'
+             'Administrator'
```
Comment (glennsarti) [inline spec/acceptance/init_spec.rb]: May not be an issue but Administrator is not guaranteed to exist e.g. on German Windows it is ` Administratoren`

Comment (glennsarti) [reply spec/acceptance/init_spec.rb]: https://tickets.puppetlabs.com/browse/PUP-6499

Comment (pmcmaw) [reply spec/acceptance/init_spec.rb]: Hey @glennsarti,

I am currently just putting in a fix for the following CI failure (FM-6924) and the specific username is only for test purposes. If you have any objections for merging please let me know.

PS - Thats really interesting, would `Administrator` differ for every language eg Japanese, Spanish, German etc?

Comment (glennsarti) [reply spec/acceptance/init_spec.rb]: I don't have any objects as I _assume_ you'll be testing on US English for the time being.  Just note that we've been bitten before.

Comment (HelenCampbell) [reply spec/acceptance/init_spec.rb]: Noted :) We'll keep an eye out.

**Pull request merged**